data/projects.js

```diff
@@ -228,4 +228,44 @@ implemented",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears", "Debugging", "pdb"]
 	}
+	{
+		"name" : "Handling languages inside other languages",
+		"desc" : "Right now we handle a single language at once for analysis. \
+If we have a php file, it is inevitable that we'd probably have chunks of \
+html inside a few of them. We probably want some method of finding specific \
+languages inside other languages and handle them accordingly. Like awk inside \
+bash, JSON inside Javascript, etc.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs.",
+			"The proposal needs to contain a clear concept.",
```

Comment:
I usually add it for proposals where the concept isn't so clear and the student needs to clear it up a bit and make some more clear things